firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/common/FirebaseCrashlyticsReportManagerTest.java

```diff
@@ -288,6 +288,21 @@ public void testOnFatalEvent_addsNoKeysToEventWhenNoneAvailable() {
     verify(logFileManager).clearLog();
   }
 
+  @Test
+  public void testOnEndSession_addsUserIdToReport() {
+    final String userId = "testUser";
+    final String sessionId = "testSessionId";
+    final long timestamp = System.currentTimeMillis();
```

Comment:
This is not problematic, per-se, but consider replacing with a constant time value to make the test deterministic

Reply:
The value of this timestamp isn't important for this test, we just need one.